pkgs/os-specific/linux/libcap/default.nix

```diff
@@ -9,19 +10,46 @@ stdenv.mkDerivation rec {
     sha256 = "0qjiqc5pknaal57453nxcbz3mn1r4hkyywam41wfcglq3v2qlg39";
   };
 
-  outputs = [ "dev" "out" ];
+  outputs = [ "dev" "lib" "doc" "bin" ]
```

Comment:
Hm, I have a recollection of the stdenv builder acting strangely if there's no `out` output. @vcunat is it still the case?